INSTALL.md

```diff
@@ -69,40 +69,36 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Ensure your database is running with the following command:
```

Comment:
This should just say "Start MYSQL with the following command". Otherwise we should include the `mysql.server status` command